src/shared/components/mutationTable/column/CosmicColumnFormatter.tsx

```diff
@@ -95,7 +95,7 @@ export default class CosmicColumnFormatter
     {
         const cosmic:CosmicMutation[]|null = CosmicColumnFormatter.getData(data, cosmicData);
 
-        let value:number = -1;
+        let value:number|undefined = -1;
```

Comment:
another question is this compilation error fix: isn't the whole project compiled at each commit? How could a compilation error creep in?

Reply:
hmm.  you should not need to allow undefined here as far as i can see.  what is compiler error?

Reply:
was old code....didn't encounter this in new PR #664